definitions/features/satellite.rb

```diff
@@ -10,6 +10,10 @@ class Features::Satellite < ForemanMaintain::Feature
     end
   end
 
+  def target_version
```

Comment:
I intend in future updates not to replicate this in multiple places but rather to use this as the canonical location and make other places reference this (such as upgrade scenario).